pymeasure/instruments/common_base.py

```diff
@@ -476,7 +476,13 @@ def fget(self,
                                maxsplit=maxsplit,
                                **values_kwargs)
             if check_get_errors:
-                errors = [str(error) for error in self.check_get_errors()]
+                try:
+                    error_list = self.check_get_errors()
+                except Exception as exc:
+                    log.error("Exception raised while getting a property with the command "
+                              f"""'{command_process(get_command)}': '{str(exc)}'.""")
+                    raise
+                errors = [str(error) for error in error_list]
```

Comment:
Is that conversion necessary? Can't we rely on the expectation that this is already a list-of-strings? Or is that a just-to-be-safe thing?

Reply:
A single error of a Scpi instrument is a list of error number and error text. The "join" command in the next line requires a list of strings. Therefore, the conversion is necessary.

Reply:
I see, thanks.